doc/obsidian.txt

```diff
@@ -343,6 +343,26 @@ The other benefit of doing this is that it will now work even if your cursor is
 on the enclosing brackets (`[[` or `]]`) or the alias part of a reference (the
 part after `|`).
 
+CUSTOMIZING THE BEHAVIOR OF FOLLOWING URL LINKS
```

Comment:
This doc file is actually generated automatically from the readme, so could you move this to readme instead?